app.js

```diff
@@ -0,0 +1,131 @@
+// Prerequisite data SIP and Access Token(Hard Coded)
+const myAccessToken = "ODcwYWE1MDQtODc5ZC00ZDhiLWI5ODYtZTk0ZGUyNTA4YzM2MTIwNmIzOTQtZDhm_PF84_1eb65fdf-9643-417f-9974-ad72cae0e10f";
+const sip = "pasoma.cisco@webex.com";
+
+document.addEventListener('DOMContentLoaded', () => {
+    const ongoingMeetingListElement = document.getElementById('ongoing-meeting-list');
+    const upcomingMeetingListElement = document.getElementById('upcoming-meeting-list');
+    const meetingDetailsElement = document.getElementById('meeting-details');
+    
+    const webhookApiUrl = 'https://webexapis.com/v1/meetings';
+    
+    async function fetchMeetings() {
+        try {
+            const response = await fetch(webhookApiUrl, {
+                headers: {
+                    'Authorization': `Bearer ${myAccessToken}`
+                }
+            });
+            const data = await response.json();
+            console.log('API Response:', data); // Log the full response
+
+            if (Array.isArray(data.items)) {
+                displayOngoingMeetings(data.items);
+                displayUpcomingMeetings(data.items);
+            } else {
+                throw new Error("Received data is not an array");
+            }
+        } catch (error) {
+            console.error('Error fetching meetings:', error);
+            ongoingMeetingListElement.innerHTML = '<p>Failed to load ongoing meetings. Please try again later.</p>';
+            upcomingMeetingListElement.innerHTML = '<p>Failed to load upcoming meetings. Please try again later.</p>';
+        }
+    }
+
+    function displayMeetingDetails(meeting) {
+        meetingDetailsElement.innerHTML = ''; // Clear previous details
+
+        const detailsHTML = `
+            <p><strong>Meeting Link:</strong> ${meeting.webLink}</p>
+            <p><strong>Meeting Number:</strong> ${meeting.meetingNumber}</p>
+            <p><strong>Host Key:</strong> ${meeting.hostKey}</p>
+            <p><strong>Password:</strong> ${meeting.password}</p>
+            <p><strong>Access Code:</strong> ${meeting.telephony.accessCode}</p>
+            <p><strong>Sip Address:</strong> ${meeting.sipAddress}</p>
+        `;

```

Comment:
Let's do this one line after another like this,

**Meeting Link**
https://cisco.webex.com/kmadavan

**Meeting Number**
543210045

**Host Key**
234567

.
.
.

**SIP Address**
kmadavan.cisco@webex.com